web/src/app/webview/src/contextManager.ts

```diff
@@ -41,7 +41,9 @@ export class ContextHost extends Mock {
 
       // Signal the necessary text changes to the embedding app, if it exists.
       if(this.oninserttext) {
-        this.oninserttext(transform.deleteLeft, transform.insert, transform.deleteRight);
+        if(transform.deleteLeft > 0 || transform.insert != '' || transform.deleteRight > 0 || transform.erasedSelection) {
```

Comment:
```typescript
import {isEmptyTransform} from '@keymanapp/keyboard-processor';
```

```suggestion
        if(!isEmptyTransform(transform) || transform.erasedSelection) {
```

I think it's probably best to keep the new condition outside of `isEmptyTransform`, but we can still use the function here.

Reply:
https://github.com/keymanapp/keyman/blob/acc38fea588986c70036571acdf2f334f83bf7b2/common/web/keyboard-processor/src/text/outputTarget.ts#L14-L19

Reply:
Thanks, didn't know about that function. I'm puzzled that passing a `null` transform is not considered empty? (seems like it would be empty on the surface)

Reply:
That's probably a reasonable change to make, based upon its current uses.  I think it's only there at present to serve as a null-guard, but returning `true` makes more sense for that.

Reply:
Cool taking that up in #11084.